src/middlewared/middlewared/service.py

```diff
@@ -1463,6 +1464,11 @@ def job_update(self, id, data):
                 extra=progress.get('extra'),
             )
 
+    @private
+    def is_boot(self):
```

Comment:
The name of this method is kind of strange. If I'm understanding your comment correctly, maybe name this something more appropriate like `def is_starting_during_boot` or something similar.